ui/advanced-widgets.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Advanced Custom Widgets
+---
+
+# Custom Widgets - Advanced Topics
+
+[[toc]]
+
+This page expands of some of the topics initially introduced in [Building Pages](building-pages.html).
+
+## Advanced Components
+
+In the custom widget system there are many options for different components to include beyond the standard OH set.  Where each component comes from determines many of the options available for configuration and styling.
+
+### F7 components
+
+The MainUI is built primarily using the [Framework7](https://framework7.io/) library.  Because most of the OH specific components are built on top of these f7 components, [the documentation of the f7 components](https://framework7.io/vue/) is useful for users wishing to understand more about the OH components as well.  In addition, however, the f7 components themselves are available as options in the widget editor.
```

Comment:
```suggestion
The MainUI is built primarily using the [Framework7](https://framework7.io/) library. 
Because most of the OH specific components are built on top of these f7 components, [the documentation of the f7 components](https://framework7.io/vue/) is useful for users wishing to understand more about the OH components as well. 
In addition, however, the f7 components themselves are available as options in the widget editor.
```

Put each sentence on its own line as this simplifies tracking changes.